### PR TITLE
1170 puma support

### PR DESCRIPTION
**Pull request by eugenk**
This is basically #1176 by @jelmd. That one was requested to be pulled into develop, not staging. This pull request fixes the destination (we can't change the destination of an existing pull request). For a description, see #1176.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -153,6 +153,9 @@ end
 group :production do
   gem 'god', '~> 0.13.4'
   gem 'exception_notification', '~> 4.0.1'
+  # a light-weight alternative for the heavy-weight, buggy Phusion apache httpd
+  # plugin. See https://github.com/puma/puma , config/puma.rb
+  gem 'puma', '~> 2.10.0'
```
Comment (jelmd) [inline Gemfile]: :-1:
I said: >= for good reasons!

Comment (0robustus1) [inline Gemfile]: No @jelmd, we'll use `~>`.

**Comment by ebolloff** [comment]
What's the solution with this? If we delete `puma` from the `Gemfile` to install it manually, we don't need to review this.

**Comment by tillmo** [comment]
yes, that is the idea, but I will discuss this with @jelmd first.

**Comment by eugenk** [comment]
@eugenk: Remove puma from the Gemfile.

**Comment by eugenk** [comment]
I just rebased staging and removed puma from the Gemfile.

**Comment by ebolloff** [comment]
:+1:

**Pull request merged**